component/src/main/java/com/celements/structEditor/DefaultStructuredDataEditorService.java

```diff
@@ -225,22 +227,27 @@ private Object getCellValue(DocumentReference cellDocRef, XWikiDocument onDoc)
     return Optional.absent();
   }
 
-  Optional<BaseObject> getCellXObject(XWikiDocument cellDoc, XWikiDocument onDoc) {
+  Optional<BaseObject> getRequestXObjectInStructEditor(XWikiDocument cellDoc, XWikiDocument onDoc) {
```

Comment:
please make it a 'private' method or if it is a service method add it to the Role interface